tests/test_implementations/test_sqldb.py

```diff
@@ -899,3 +901,11 @@ def test_transitive_object_properties(self):
 
     def test_simple_subproperty_of_chains(self):
         self.compliance_tester.test_simple_subproperty_of_chains(self.oi)
+
+    def test_entities_metadata_statements(self):
```

Comment:
You can move this into the core compliance tests so it can be reused by other implementations

Reply:
Done!